src/main/scala/is/hail/stats/ComputeRRM.scala

```diff
@@ -58,3 +37,42 @@ object ComputeRRM {
     }
   }
 }
+
+// each row has mean 0, norm sqrt(n), variance 1, constant variants are dropped
+object ToNormalizedRowMatrix {
+  def apply(vds: VariantDataset): RowMatrix = {
+    val n = vds.nSamples
+    val rows = vds.rdd.flatMap { case (v, (va, gs)) => RegressionUtils.toNormalizedGtArray(gs, n) }.map(Vectors.dense)
+    val m = rows.count()
+    new RowMatrix(rows, m, n)
+  }
+}
+
+// each row has mean 0, norm sqrt(n), variance 1, constant variants are dropped
+object ToNormalizedIndexedRowMatrix {
+  def apply(vds: VariantDataset): IndexedRowMatrix = {
+    val n = vds.nSamples
+    val variants = vds.variants.collect()
+    val variantIdxBc = vds.sparkContext.broadcast(variants.index)
+    val indexedRows = vds.rdd.flatMap { case (v, (va, gs)) => RegressionUtils.toNormalizedGtArray(gs, n).map(a => IndexedRow(variantIdxBc.value(v), Vectors.dense(a))) }
+    new IndexedRowMatrix(indexedRows, variants.length, n)
+  }
+}
+
+// each row has mean 0, norm approx sqrt(n), variance approx 1, constant variants are included as zero vector
+object ToHWENormalizedIndexedRowMatrix {
```

Comment:
The old ToStandardizedIndexedRowMatrix used nNonRefAlleles but you use gt directly.  Did anything depend on the old behavior?